submissions/SamVal007/Memory_Pair_Game/main.js

```diff
@@ -0,0 +1,134 @@
+let similarCards = [];
+let blockedClick = false;
+const images = ['Dart_Weider', 'lyuk-skajuoker', 'R2-D2', 'c-3po',
+                'printsessa-leya-organa', 'enakin-skajuoker', 'obi-van-kenobi',
+                 'dart-sidius', 'kvaj-gon-dzhinn','joda'];
+
+const mainAttributes = {
+    container: 'container',
+    game: 'game',
+    card: 'card',
+    front: 'front_card',
+    frontImg: 'img-front_card',
+    back: 'back_card',
+    backImg: 'img-back_card',
+    flipped: 'flipped',
+    hidden: 'hidden'
+};
+
+const cleanCountArray = () => similarCards = [];
+
+const flippSelectedCards = () => {
+    setTimeout(() => {
+        document.querySelectorAll(".flipped").forEach((card) => {
+            card.classList.remove(mainAttributes.flipped);
+            cleanCountArray();
+            blockedClick = false;
+        });
+    }, 900);
+};
+
+const hideSimilarCards = () => {
+    setTimeout(() => {
+        document.querySelectorAll(".flipped").forEach((card) => {
+            card.classList.add(mainAttributes.hidden);
+            cleanCountArray();
+            blockedClick = false;
+            const counterHiddenCards = document.querySelectorAll(".hidden").length;
+            const amountOfCards = (images.length * 2);
+            console.log(amountOfCards);
+            if (counterHiddenCards == amountOfCards) {
+                victoryMessage();
+            }
+        });
+    }, 600);
+};
+
+const compareSimilarCards = () => {
+    const countCards = similarCards.length;
+    if (countCards == 2) {
+        blockedClick = true;
+        if (similarCards[0] === similarCards[1]) {
```

Comment:
for this one use destructuring please
example 
```
const [firstCard, secondCard] = similarCards;
if(...)
```

Reply:
@SamVal007 not fixed

Reply:
Yep, sorry, It's still not fixed.
Gonna fix this and other things today.
Thank you!

Reply:
Thank you!
Done